OptimoveSDK/build.gradle

```diff
@@ -16,6 +16,10 @@ allprojects {
     repositories {
         mavenCentral()
         google()
+        maven {
+            url 'http://developer.huawei.com/repo/'
+            allowInsecureProtocol = true
```

Comment:
why http?

Reply:
Ah this should actually work over https, can change & test (at least I did change it when I was tinkering recently and it was fine).

Reply:
i think it might be no more than a typo in our docs o_O